roles/ansible/tower/config-ansible-tower-ocp/README.md

```diff
@@ -18,8 +18,9 @@ The variables used to install Ansible Tower on OpenShift are outlined in the tab
 
 | Variable | Description | Required | Defaults |
 |:---------|:------------|:---------|:---------|
-|ansible_tower_download_url| URL of Ansible Tower installer artifact repository|no|https://releases.ansible.com/ansible-tower| 
-|ansible_tower_version| Version of Ansible Tower Openshift installer|no|3.8.1-1|
+|ansible_tower_download_url|URL of Ansible Tower installer artifact repository|no|https://releases.ansible.com/ansible-tower/setup_openshift/ansible-tower-openshift-setup-{{ ansible_tower_version }}.tar.gz|
```

Comment:
This looks off in the rendered markdown.

The default could be enclosed in backticks here to avoid having it be auto converted to a link and cutting off at the double curly braces.